docker-compose-pg-load.yml

```diff
@@ -0,0 +1,50 @@
+---
+ # docker-compose -f docker-compose-pg-load.yml up postgres-load
+version: '3.7'
+
+services:
+  postgres-pgmonitor:
```

Comment:
maybe it's better to include it to default docker-compose and add some instructions how to use it.

Reply:
I can't understand how we are going to use it. What's use case?

Reply:
Usage is docker-compose -f docker-compose-pg-load.yml up postgres-load. Then let it running and it generating required fields in pg_stat_monitor. Add to monitoring and see new metrics filled here. It is same how Andrii did it, I just separate service pg with monitor to this file to avoid problems with test.